test/general/openpgp.js

```diff
@@ -640,8 +709,8 @@ module.exports = () => describe('OpenPGP.js public api tests', function() {
       await expect(privateKey.decrypt('wrong passphrase')).to.eventually.be.rejectedWith('Incorrect key passphrase');
     });
 
-    it('Decrypting key with correct passphrase returns true', async function () {
-      expect(await privateKey.decrypt(passphrase)).to.be.true;
+    it('Decrypting key with correct passphrase does not throw', async function () {
+      await expect(privateKey.decrypt(passphrase)).to.not.be.rejected;
```

Comment:
```suggestion
    it('Can decrypt key with correct passphrase', async function () {
      await privateKey.decrypt(passphrase);
```

Reply:
I know it's redundant, but in cases such as this one I would not mind being a bit declarative to explicitly mark what we are testing.